packages/prettier-plugin/src/printer.ts

```diff
@@ -187,6 +188,7 @@ export function createSquigglePrinter(
               softline,
               ")",
             ]),
+            node.value.returnUnitType ? typedPath(node).call(print, "value", "returnUnitType") : "",
```

Comment:
@berekuk This line of code (and two other similar lines) are giving TypeScript errors:

> Argument of type '(path: AstPath<SquiggleNode>) => Doc' is not assignable to parameter of type 'CallCallback<NodeTypeSignature | undefined, Doc>'.

Apparently I need to coerce the type to non-undefined but this line is doing some weird dynamic function call thing that I don't understand, so I don't know how to fix it. Do you know how to fix this?

Reply:
I just put this as `@ts-ignore` for now since it didn't seem that important